### ISSUE TITLE
review and add new t&c , privacy policies

### ISSUE THREAD

**Issue by sasha-astiadi** [threefoldfoundation/info_legal#18]
https://library.threefold.me/info/legal/#/
the last time it was updated was last year (2023).

- [ ] please review the current clauses, legal addresses, and terms, and edit accordingly.

also 

- [ ] please create a new t&c, disclaimer, privacy policies, for TF9 

- [ ] please create a new t&c, disclaimer, privacy policies, for OurWorld Holdings

since they are all separate entities with different legal addresses and registry.

you can simply create one on google docs and i can help putting it into this wiki.

**Comment by samtaggart** [comment]
I agree here big time, but we are getting rid of library.threefold.me

So please review what is on the manual instead:

https://manual.grid.tf/knowledge_base/legal/terms_conditions_all3.html

It was pulled directly from the library so it should all be the same.

Once this is updated we can copy it to https://docs.threefold.info/docs/category/legal as well.

**Comment by Karoline888** [comment]
Matthew Christensen will support on this task and I follow what Sam suggests, he will start to review the manual.
Once it is clean there we can copy it to [https://docs.threefold.info/docs/category/legal](https://docs.threefold.info/docs/category/legal)

Sasha and me will jump on a quick call with Matthew to clarify the story/ task and also how he can make suggestions on the manual. 

Second step will be to create a new t&c, disclaimer, privacy policies, for TF9

Third step will be create a new t&c, disclaimer, privacy policies, for OurWorld Holdings

**Comment by samtaggart** [comment]
Great! Super helpful and super needed.

**Comment by sasha-astiadi** [comment]
### Problem Summary

Currently, different websites and companies do not have their own terms and conditions (T&Cs), we are using the ThreeFold privacy policy and T&Cs for all of them. This is incorrect for sites and companies that do not belong to ThreeFold.

### Expected Solution

- Create separate legal documents for each entity.
- Specifically, ThreeFold, OurWorld, and TF9 should each have their own T&Cs and privacy policies.
- The legal documents should be hosted and written separately for each brand to avoid any confusion or legal inconsistency.

### Additional Notes

- The existing ThreeFold legal content should **not** be applied to entities that are not directly part of ThreeFold (e.g., OurWorld, TF9).
- For creating this legal content, someone with expertise in drafting legal documents should be involved. I'm not the right person for the content itself but am available to help with implementation once it's ready.
- we should then create :
1. docs.ourworld.info/docs/category/legal
2. docs.tf9.info/docs/category/legal

### References
- [Current ThreeFold Legal Docs](https://docs.threefold.info/docs/category/legal)
- [List of active websites](https://docs.google.com/spreadsheets/d/1RsCcmXt2ds4wWK-Tdp6KokyoGl-dEW96W84O89eiE4U/edit?usp=sharing)

**Comment by Karoline888** [comment]
Any news here @Matthewmarkc ?

**Comment by Matthewmarkc** [comment]
Sam followed up with me on this. It’s a low risk but very time consuming task so it’s back- burnered for now.
